lib/glob.js

```diff
@@ -1,12 +1,19 @@
 'use strict';
 
-var braces = require('braces');
-var brackets = require('expand-brackets');
-var extglob = require('extglob');
-var parse = require('parse-glob');
+var isExtglob = require('is-extglob');
 var chars = require('./chars');
 
 /**
+ * Lazily required module dependencies.
+ */
+
+var lazy = require('lazy-cache')(require);
+var lazyBraces = lazy('braces');
```

Comment:
Why bother changing all the var names for the modules? Isn't part of the purpose of `lazy-cache` that it be seamless?

Totally a style type of comment, an argument could easily be made the other way that they might as well be explicit.